### DISCUSSION TITLE
`vercel.sh` vs. `vercel.app`

### DISCUSSION THREAD

**Discussion by borekb** [vercel/vercel#6991] [General] [answered]
I've noticed several demos running on `vercel.sh` instead of `vercel.app` but could not find any information around it, is this is a new domain that's going to replace `vercel.app`? Will it live side by side? Thanks for clarifying.

**Comment by leerob** [comment, marked as answer]
Hey! `.vercel.sh` is a domain that we (Vercel) own. Any demos you see here are from the Vercel team. `.vercel.app`, which is what Vercel users have access to, will not change 😄
